concepticondata/conceptlists/Zhang-2019-308.tsv

```diff
@@ -0,0 +1,309 @@
+ID	NUMBER	ENGLISH	GLOSS_IN_SOURCE	CONCEPTICON_ID	CONCEPTICON_GLOSS	REMARK
+Zhang-2019-308-1	1	diminutive suffix	(diminutive suffix)			
+Zhang-2019-308-2	2	the brown bear	Asiatic brown bear	1820	BEAR	"This word could generally mean ""bear"", as there are not many types of bears to the local people"
+Zhang-2019-308-3	3	I	I, my	1209	I	
+Zhang-2019-308-4	4	after	after	1685	AFTER	
+Zhang-2019-308-5	5	the afternoon	afternoon	1807	AFTERNOON	
+Zhang-2019-308-6	6	the Allium tuberosum	allium			
+Zhang-2019-308-7	7	the arm	arm	1673	ARM	
+Zhang-2019-308-8	8	the armpit	armpit	1886	ARMPIT	
+Zhang-2019-308-9	9	the arrow	arrow	977	ARROW	
+Zhang-2019-308-10	10	the axe	axe	677	AXE	
+Zhang-2019-308-11	11	the bark	bark (noun)	1204	BARK	
+Zhang-2019-308-12	12	to be curved	be bent, curved	1698	CURVED	
+Zhang-2019-308-13	13	black	be black	163	BLACK	
+Zhang-2019-308-14	14	cloudy	be cloudy, dark	2341	CLOUDY	
+Zhang-2019-308-15	15	hard	be hard	1884	HARD	
+Zhang-2019-308-16	16	loud	be loud	377	LOUD	
+Zhang-2019-308-17	17	new	be new	1231	NEW	
+Zhang-2019-308-18	18	tall	be tall	711	TALL	
+Zhang-2019-308-19	19	the beak	beak	73	BEAK	
+Zhang-2019-308-20	20	the bear	bear	1820	BEAR	
+Zhang-2019-308-21	21	the bee	bee	665	BEE	
+Zhang-2019-308-22	22	the belly	belly	1251	BELLY	
+Zhang-2019-308-23	23	the boat	boat	1844	BOAT	
+Zhang-2019-308-24	24	the body, person	body, person	1480	BODY	
+Zhang-2019-308-25	25	the body, self	body, self	1480	BODY	
+Zhang-2019-308-26	26	the brain	brain	1583	BRAIN	
+Zhang-2019-308-27	27	the broad bean	broad bean	2574	BEAN OR PEA	I think this one could be regarded as a general term
+Zhang-2019-308-28	28	the brown-and-white bear	brown-and-white bear	1820	BEAR	
+Zhang-2019-308-29	29	the chicken	chicken	1318	CHICKEN	
+Zhang-2019-308-30	30	the child	child	2099	CHILD	
+Zhang-2019-308-31	31	the claw	claw	72	CLAW	
+Zhang-2019-308-32	32	the clothes	clothes	1895	CLOTHES	
+Zhang-2019-308-33	33	the colour, countenance	colour, countenance	1613	COLOR	
+Zhang-2019-308-34	34	the command	command	1998	COMMAND	
+Zhang-2019-308-35	35	the cooked rice	cooked rice	1191	COOKED RICE	
+Zhang-2019-308-36	36	the cow	cow	1007	COW	
+Zhang-2019-308-37	37	to cry (of birds or animals)	cry (of birds or animals)	1839	CRY	
```

Comment:
CRY 1839 refers to the act of shedding tearing under the influence of emotions. This is clearly a different concept.